File: DSA/LeetCode/Arrays/1470. Shuffle the Array/Solution.js
/**
 * @param {number[]} nums
 * @param {number} n
 * @return {number[]}
 */
var shuffle = function (nums, n) {
  const len = nums.length
  const arr = []

  for (let i = 0; i < n; i++) {
    arr.push(nums[i])
    arr.push(nums[i + n])
  }
  return arr
};

File: DSA/LeetCode/Arrays/1470. Shuffle the Array/Solution.py
class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        new_arr = []
        i = 0

        while i < n:
            new_arr.append(nums[i])
            new_arr.append(nums[n+i])
            i += 1
        return new_arr
